Configs/.local/share/bin/keybinds_hint.sh

```diff
@@ -89,7 +90,7 @@ while [ "$#" -gt 0 ]; do
   shift
   kb_hint_line="$1"
   ;;
-  -* | --help) # Add Help message
+  --help) # Add Help message
```

Comment:
can you make it only

` -* ` 

It is useful to catch other errors. (Ignores shellcheck because it would spit out warnings)

Reply:
Personally think `--help` is solid, but changed as you wanted, don't know what to do in the _tip_ then(isn't--help anymore)

Reply:
😁 --help just to make it look conventional.  But it would still fall back to "help" if the users tried to input anything ` -* ` . Instead of throwing errors.